catalog/core/catalog-core-api/src/main/java/org/codice/alliance/catalog/core/api/types/Isr.java

```diff
@@ -27,6 +27,21 @@
     String CLOUD_COVER = "isr.cloud-cover";
 
     /**
+     * Attribute name for accessing the snow cover existence for this Metacard. <br/>
+     */
+    String SNOW_COVER = "isr.snow-cover";
+
+    /**
+     * Attribute name for accessing the minimum snow depth (centimeters) for this Metacard. <br/>
+     */
+    String SNOW_DEPTH_MIN = "isr.snow-depth-min";
```

Comment:
Recommend adding units to these constants; e.g. `SNOW_DEPTH_MIN_CENTIMETERS = isr.snow-depth-min-centimeters`.